pkg/framework/plugins/defaultevictor/types.go

```diff
@@ -35,4 +35,5 @@ type DefaultEvictorArgs struct {
 	PriorityThreshold       *api.PriorityThreshold `json:"priorityThreshold"`
 	NodeFit                 bool                   `json:"nodeFit"`
 	MinReplicas             uint                   `json:"minReplicas"`
+	MinPodAge               uint                   `json:"minPodAge"`
```

Comment:
What about to use `*time.Duration` instead? Users can then decide which unit they want to choose.

Reply:
Thanks for the comment! I did the change here https://github.com/kubernetes-sigs/descheduler/pull/1475/commits/ec7ad04c51929cb50cb6a946df63212c4c80154c . Let me know if thats what you expect! ty

Reply:
This works. Thank you